spec/casbin/persist/adapters/file_adapter_spec.rb

```diff
@@ -1,159 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Why do we delete this tests?

Reply:
overlapped by tests from spec/casbin/enforcer_spec.rb
And doesn't work with stubs

Reply:
Is saving of a policy to the file tested?